src/parse.h

```diff
@@ -329,6 +329,7 @@ struct net_definition {
         char *remote_ip;
         char *input_key;
         char *output_key;
+	guint ttl;
```

Comment:
Please fix the indentation to be in line with the other variables of this struct (8 spaces).